app/talent/events/page.js

```diff
@@ -190,6 +187,103 @@ export default function Events() {
           </Box>
         </Flex>
       </Box>
+
+      {events.map((event) => {
+  const startDate = event.EventDate ? new Date(event.EventDate) : null;
+
+  const formattedDate = startDate
+    ? `${startDate.toLocaleString("default", { month: "short" })} ${startDate.getDate()}`
+    : "N/A";
+
+  return (
+    <Box
```

Comment:
Can we move this into a separate component called `EventItem` within the /components folder?